deploy/csi-daemonset.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  namespace: mayastor
+  name: mayastor-csi
+  labels:
+    openebs/engine: mayastor
+spec:
+  selector:
+    matchLabels:
+      app: mayastor-csi
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  minReadySeconds: 10
+  template:
+    metadata:
+      labels:
+        app: mayastor-csi
+    spec:
+      nodeSelector:
```

Comment:
Don't we need hostnetwork: true here? as iscsiadm will run on the node which is running mayastor csi plugin/

Reply:
thanks for reminding me! 😅  I had this change in a different workspace and accidentally did not include it in final PR ..